### PR TITLE
Add Show/Hide password emblem in password entry

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: src/gs-lock-plug.c [hunk 1]
```diff
@@ -2103,6 +2103,28 @@ on_note_text_buffer_changed (GtkTextBuffer *buffer,
 	gtk_widget_set_sensitive (plug->priv->note_ok_button, len <= NOTE_BUFFER_MAX_CHARS);
 }
 
+static void
+entry_icon_clicked (GtkEntry       *entry,
+                    gint            position,
+                    GdkEventButton *event,
+                    gpointer        data)
```
Comment (raveit65) [inline src/gs-lock-plug.c:2110]: `-Wunused-parameter` warnings.
https://travis-ci.org/github/mate-desktop/mate-screensaver/jobs/736309344#L2604
```
gs-lock-plug.c:2109:37: warning: unused parameter 'event' [-Wunused-parameter]

                    GdkEventButton *event,

                                    ^

gs-lock-plug.c:2110:37: warning: unused parameter 'data' [-Wunused-parameter]

                    gpointer        data)
```

**Comment by raveit65** [comment]
Beside from the build warnings it works fine.

File: src/gs-lock-plug.c (hunk 1 above)
Comment (rbuj) [reply src/gs-lock-plug.c:2110]: done

**Review by raveit65 — APPROVED** [review]
Works fine with double-click event now and warning is fixed.

**Review by lukefromdc — APPROVED** [review]
Didn't think to test the tooltip but the icon works and shows up. Subtle with my GTK theme but definately works

**Pull request merged**